lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 crate-type = ["staticlib"]
 
 [dependencies]
-bitflags = "1.3.2"
+bitflags = "2.0.2"
```

Comment:
Same as last week - we should do this one in a separate PR, it's a big change with possible breakages.

@rosstimothy, what did we decide for this again?

Reply:
@ibeckermayer can you have a look and see what changes are necessary to get on `bitflags` v2? If it's not trivial, we can just add this to the ignore list.